src/Sentry/Protocol/SpanStatus.cs

```diff
@@ -0,0 +1,78 @@
+using System.Runtime.Serialization;
+
+namespace Sentry.Protocol
+{
+    /// <summary>
+    /// Span status.
+    /// </summary>
+    public enum SpanStatus
+    {
+        /// <summary>The operation completed successfully.</summary>
+        [EnumMember(Value = "ok")]
```

Comment:
Oh I thought you killed these `System.Runtime.Serialization` attributes

Reply:
I decided to leave these and later make it so they're picked up by the new serialization

Reply:
Currently they are ignored